hypothesis-python/src/hypothesis/strategies/_internal/misc.py

```diff
@@ -51,9 +51,6 @@ def __repr__(self):
             return "none()" + suffix
         return f"just({get_pretty_function_description(self.value)}){suffix}"
 
-    def calc_has_reusable_values(self, recur):
-        return True
-
```

Comment:
`JustStrategy` is a subclass of `SampledFromStrategy`, so we can just delegate to the (now-fixed) superclass implementation.